llama_stack/distributions/ci-tests/build.yaml

```diff
@@ -25,6 +25,7 @@ distribution_spec:
     - provider_type: inline::milvus
     - provider_type: remote::chromadb
     - provider_type: remote::pgvector
+    - provider_type: remote::mongodb
```

Comment:
you need to also add an image that'll run MongoDB and expose the vector search functionality.

Reply:
you'll need to add this to the `./github/workflows/integratino-vector-io.tests.yaml` file like here: https://github.com/llamastack/llama-stack/blob/main/.github/workflows/integration-vector-io-tests.yml#L43